Q_hub/static/style.css

```diff
@@ -0,0 +1,83 @@
+nav{
```

Comment:
Well, you don't have to look for anything , it's just the common sense that you could have used for including only the backend files. Don't use `git add .` for adding the files instead add the backend files manually & there are hardly 5-6 files. Moreover, frontend files don't affect the merging process, its just to make the merge commits clean and specific to the issue, so that in future if we have to revert then we can easily revert back.